pgx-pg-sys/build.rs

```diff
@@ -536,14 +536,26 @@ fn run_bindgen(pg_config: &PgConfig, include_h: &PathBuf) -> eyre::Result<syn::F
         .header(include_h.display().to_string())
         .clang_arg(&format!("-I{}", includedir_server.display()))
         .parse_callbacks(Box::new(PgxOverrides::default()))
+        .allowlist_file(".*confdefs.h")
+        .allowlist_file(".*([fF]uncs|mgr).h")
+        .allowlist_file(".*htup(_details)?.h")
+        .allowlist_file(".*syscache.h")
+        .allowlist_file(".*mcxt.h")
+        .allowlist_file(".*(storage|catalog|access|commands|executor|adt|optimizer|rewrite|postmaster|tcop|replication|nodes|postgres|parse|pg_|item|heap).*")
```

Comment:
This effectively includes all of the source directories that Postgres uses (since many files will be `#include`d and thus regexed as "path/file.h") and a handful of keywords that are common in Postgres source filenames.

Note that we cannot simply use `blocklist_file` on libc header names, I tried that. They're too :sparkles: Special :sparkles:. But also we cannot simply use `allowlist_file` on `.*include.*` because then all of glibc appears again.